apiserver/worker/backend.py

```diff
@@ -6,6 +6,9 @@
 from time import gmtime, strftime
 
 
+MAX_BOT_UPLOAD_SIZE = 100 * 1024 * 1024
```

Comment:
Might want to leave a note here and in `apiserver/worker/config.py` near `MAX_COMPILED_BOT_UPLOAD_SIZE` that the values should match.